lib/tasks/rollover.rake

```diff
@@ -0,0 +1,12 @@
+namespace :rollover do
+  desc "Rollover providers, courses and locations to the next cycle"
+  task :providers, [:provider_codes] => :environment do |_task, args|
+    provider_codes = args[:provider_codes].split " "
```

Comment:
How about when you run just `rake rollover:providers`? This will error at the moment.

Reply:
Fixed